### PR TITLE
[MERGE 9/13] feat(lookup tables): Add CSV Value Parsing section

### PR DESCRIPTION
**Pull request by ccobb-nr**
Ready for review - please merge this at a time that it will get applied to the public docs no earlier than 2023-09-13. 

Previously, when querying NRQL Lookup, an attempt was made to interpret values as numbers, strings, or boolean values.  This update defines the rules behind the parsing of table values.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/logs/ui-data/lookup-tables-ui.mdx [hunk 1]
```diff
@@ -27,33 +27,204 @@ For examples of queries using lookups, see [How to query lookup table data](/doc
...
+              * Maximum integer supported: 9223372036854775807 (Note, numbers higher than 9007199254740991 may lose precision when viewed in a web browser)
+              * Minimum integer supported: -9223372036854775808 (Note, numbers lower than -9007199254740991 may lose precision when viewed in a web browser)
+              * When interpreting a float value, the integer part of the value (the part before the decimal in the original string value) must lie in the maximum/minimum supported integer bounds.
+              * If the value contains a decimal, there must be at least one numeric digit before and after the decimal.
+              * Signed values:
+              * Values that start with a numeric digit or a plus "+" sign are considered positive numbers.
+              * Values that start with a minus "-" sign or are surrounded in parentheses are considered negative numbers.
+              * Original values are never trimmed. Whitespace at the beginning or end of a value will cause it not to be parsed to a boolean or numeric value.
```
Comment (mlobiondo-newrelic) [inline src/content/docs/logs/ui-data/lookup-tables-ui.mdx:88]: I'd say, "...will cause it to be parsed as a string."

File: src/content/docs/logs/ui-data/lookup-tables-ui.mdx [hunk 2]
```diff
@@ -27,33 +27,204 @@ For examples of queries using lookups, see [How to query lookup table data](/doc
...
+            string<br/>
+            string
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <b>Double Quotes</b>
+            * Any value can be wrapped in double quotes. Double quotes are required if the value contains a comma or starts with a double quotes.
+            * Within a double-quoted value, two double quotes in a row will parse to a single double quote in the resulting value.
+            * Wrapping a value in does not force it to be interpreted as a string. Parsing of the value still occurs.
```
Comment (mlobiondo-newrelic) [inline src/content/docs/logs/ui-data/lookup-tables-ui.mdx:129]: Wrapping a value in _double quotes_

File: src/content/docs/logs/ui-data/lookup-tables-ui.mdx [hunk 3]
```diff
@@ -27,33 +27,204 @@ For examples of queries using lookups, see [How to query lookup table data](/doc
...
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>
+            <b>Value Types</b>
+            * String: This is the default type for any value, unless criteria are met to interpret it as a number or boolean.
+            * Boolean: If the value is equal to true or false, case-insensitive, the value is parsed as a boolean.
+            * Numbers: Value parsing can interpret integer and float values, with the following stipulations:
+              * Scientific notation is supported. Values included scientific notation are parsed to a float value.
```
Comment (mlobiondo-newrelic) [inline src/content/docs/logs/ui-data/lookup-tables-ui.mdx:79]: Values _including_?

File: src/content/docs/logs/ui-data/lookup-tables-ui.mdx [hunk 4]
```diff
@@ -27,33 +27,204 @@ For examples of queries using lookups, see [How to query lookup table data](/doc
...
+              * If the numeric value contains a decimal or the letter E, it will parse to a float value. Otherwise, it will parse to an integer value.
+              * Maximum integer supported: 9223372036854775807 (Note, numbers higher than 9007199254740991 may lose precision when viewed in a web browser)
+              * Minimum integer supported: -9223372036854775808 (Note, numbers lower than -9007199254740991 may lose precision when viewed in a web browser)
+              * When interpreting a float value, the integer part of the value (the part before the decimal in the original string value) must lie in the maximum/minimum supported integer bounds.
```
Comment (mlobiondo-newrelic) [inline src/content/docs/logs/ui-data/lookup-tables-ui.mdx:83]: I'd say "...must lie _within_ the max..."

**Review by bradleycamacho — APPROVED** [review]

**Pull request merged**